_tools/riot-stats.sh

```diff
@@ -0,0 +1,33 @@
+#!/bin/bash
+
+# Parse RIOT statistics and dump them
+#
+# Usage:
+#  $ ./riot-stats.sh repo
+# * repo: path to RIOT's repository
+
+RIOTBASE=$1
+
+countboards () {
+  make -C ${RIOTBASE} info-boards | tr " " "\n" | wc -l | xargs printf "boards: %d\n" 
+}
+
+countcpus () {
+  cd $RIOTBASE
+  git grep "config CPU_FAM_" | grep "Kconfig" | wc -l | xargs printf "cpus: %d\n"
+  cd - > /dev/null
```

Comment:
```suggestion
  git -C ${RIOTBASE} grep "config CPU_FAM_" | grep "Kconfig" | wc -l | xargs printf "cpus: %d\n"
```
Same option could be used below

Reply:
just for the record: older git software versions do not support `-C` parameter. this is the reason why it was not used. i don't know when exactly the parameter was introduced, and whether we can safely assume that it runs out of the box.

Reply:
> older git software versions do not support -C parameter

They might be very old. This option was [used by git itself for quite some time now](https://github.com/git/git/commit/11ca4bec96e00f7ed18dbd2a475a8d568f4b85f3). The [original commit](https://github.com/git/git/commit/087350398e8b2c5d4b39f051b23a2e533f4d830b) is from 2013.

Reply:
The official [git doku](https://git-scm.com/docs/git/) lists the `-C` parameter since [version 2.1.4](https://git-scm.com/docs/git/2.1.4) which was released [December 2014](https://github.com/git/git/tree/v2.1.4) (and it is probably supported longer, the current doc just doesn't go back that far). Even Debian stretch (Debian's current old stable) has [git 2.11](https://packages.debian.org/stretch/all/git-all/download). Do we really need to provide legacy support that old for our website?

Reply:
TBH, I saw this was already [used somewhere else in this PR](https://github.com/RIOT-OS/riot-os.org/pull/17/files#diff-76ed074a9305c04054cdebb9e9aad2d818052b07091de1f20cad0bbac34ffb52R39), so just asked for this pattern to be applied everywhere. Since it avoids all these directory round-trips.

Reply:
>  Do we really need to provide legacy support that old for our website?

I don't think so. +1 for `-C`.

Reply:
thanks for checking @miri64 .